run_test.sh

```diff
@@ -19,7 +19,7 @@ pytest tests/dag_validation_test/ -p no:cacheprovider -s --disable-warnings
 
 echo "running pylint"
 PYLINTHOME=/tmp/datahub-dags-pylint \
- pylint tests/ peerscout/
+ pylint tests/ peerscout/ --init-hook="import sys; sys.setrecursionlimit(1500)"
```

Comment:
did you encounter an issue with a particular test?

Reply:
pylint running into recursion errors while building AST because of this line https://github.com/elifesciences/peerscout-dags/blob/generalized_data_source/peerscout/utils/bq_data_service.py#L137